File: app/src/main/java/localhost/android/me/model/ProviderModel.java
package localhost.android.me.model;

public class ProviderModel
{
    private long id;
    private String code;
    private String name;
    private long countryId;
    private String countryIso;
    private String countryName;
    private boolean custom;

    public ProviderModel(String code, String countryIso, String name, String custom)
    {
        this.code = code;
        this.countryIso = countryIso;
        this.name = name;
        this.custom =  Boolean.parseBoolean(custom);
    }

    public long getId()
    {
        return this.id;
    }

    public String getCode()
    {
        return this.code;
    }

    public String getName()
    {
        return this.name;
    }

    public String getFullTitle()
    {
        return "[" + this.countryIso + "] " + this.name;
    }

    public long getCountryId()
    {
        return this.countryId;
    }

    public String getCountryIso()
    {
        return this.countryIso;
    }

    public String getCountryName()
    {
        return this.countryName;
    }

    public boolean isCustom()
    {
        return this.custom;
    }

    public void setId(long id)
    {
        this.id = id;
    }

    public void setCountryId(long countryId)
    {
        this.countryId = countryId;
    }

    public void setCountryName(String countryName)
    {
        this.countryName = countryName;
    }
}

File: app/src/main/java/localhost/android/me/model/ProviderModel.kt
package localhost.android.me.model

class ProviderModel(
    private var code: String,
    private var countryIso: String,
    private var name: String,
    custom: String
)
{
    private var id: Long = 0
    private var countryId: Long = 0
    private var countryName: String = ""
    private var custom: Boolean = custom.toBoolean()

    fun getId(): Long
    {
        return this.id
    }

    fun getCode(): String
    {
        return this.code
    }

    fun getName(): String
    {
        return this.name
    }

    fun getFullTitle(): String
    {
        return "[" + this.countryIso + "] " + this.name
    }

    fun getCountryId(): Long
    {
        return this.countryId
    }

    fun getCountryIso(): String
    {
        return this.countryIso
    }

    fun getCountryName(): String
    {
        return this.countryName
    }

    fun isCustom(): Boolean
    {
        return this.custom
    }

    fun setId(id: Long)
    {
        this.id = id
    }

    fun setCountryId(countryId: Long)
    {
        this.countryId = countryId
    }

    fun setCountryName(countryName: String)
    {
        this.countryName = countryName
    }
}
